delete-review-app/action.yml

```diff
@@ -0,0 +1,140 @@
+name: Delete Review App
+description: Delete Review App
+inputs:
+  azure-credentials:
+    description: A JSON string containing service principal credentials.
+    required: true
+  gcp-wip:
+    description: The full identifier of the GCP Workload Identity Provider.
+    required: false
+  gcp-project-id:
+    description: The name of the GCP Project ID.
+    required: false
+  environment:
+    description: Environment to delete from
+    default: review
+    required: false
+  github-event-name:
+    description: github event
+    default: pull_request
+    required: false
+  global-config:
```

Comment:
hardcode global-config as review and remove input

Reply:
removed